plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -44,6 +44,38 @@ public function __invoke( OD_Tag_Visitor_Context $context ): bool {
 		return false;
 	}
 
+	/**
+	 * Computes responsive sizes for the current element based on its boundingClientRect width captured in URL Metrics.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param OD_Tag_Visitor_Context $context Context.
+	 * @return non-empty-string[] Computed sizes.
+	 */
+	private function compute_sizes( OD_Tag_Visitor_Context $context ): array {
```

Comment:
Nit-pick: Can we move this method further down in the file, as it's a very specific helper method called by the higher-level `process_img()` private method?